modules/dists/StencilDist.chpl

```diff
@@ -122,6 +107,95 @@ config param disableStencilLazyRAD = defaultDisableLazyRADOpt;
 // dataParMinGranularity: the minimum required number of elements per
 //                        task created
 //
+
+/*
+
+  .. warning::
+
+    As of 1.13 the Stencil distribution is neither heavily documented nor tested.
```

Comment:
Should we mention also the long-term goal of merging stencil dist functionality back into block dist?